src/data/people.json

```diff
@@ -109,6 +109,16 @@
     "orgID": "Y2lzY29zcGFyazovL3VzL09SR0FOSVpBVElPTi9jb25zdW1lcg",
     "status": "self"
   },
+  "noStatus": {
```

Comment:
`default` object is supposed to represent the "no status" state. I don't think this object is necessary